app/styles/partials/_jsonVars.scss

```diff
@@ -1,4 +1,6 @@
 $grid-w-1:195px;
 $grid-w-2:400px;
 $grid-h-1:170px;
-$grid-h-2:350px;
\ No newline at end of file
+$grid-h-2:350px;
+$numOfSurvivalCurveLimit:20;
```

Comment:
This is automatically generated based on configs.json As long as you define a variable in there, there will be a variable in _jsonVars.scss